clarifai/datasets/export/inputs_annotations.py

```diff
@@ -61,8 +61,11 @@ def __init__(self,
   def _download_temp_archive(self, archive_url: str,
                              chunk_size: int = 128) -> tempfile.TemporaryFile:
     """Downloads the temp archive of InputBatches."""
-    session = requests.Session()
-    r = session.get(archive_url, stream=True)
+    r = self.session.get(archive_url, stream=True)
+    if r.headers['content-type'] == 'application/json':
```

Comment:
Please define constants for content-type i.e. `'application/zip'` and `'application/json'` in a module (e.g. `http.py`) in the [clarifai.constants](https://github.com/Clarifai/clarifai-python/tree/master/clarifai/constants) package